man/man8/zfs-send.8

```diff
@@ -140,6 +141,25 @@ If the
 flag is used to send encrypted datasets, then
 .Fl w
 must also be specified.
+.It Fl X , -exclude Ar pool/dataset Ns Oo , Ns Ar pool/dataset Oc Ns ...
+When the
+.Fl R
+flag is given,
+.Fl X
+can be used to specify a list of datasets to be excluded from the
+data stream.
+The
+.Fl X
+option can be used multiple times, or the list of datasets can be
+specified as a comma-separated list, or both.
+Each dataset must be specified as
+.Em pool/dataset ,
+and will exclude all children datasets as well.
+The
+.Fl R
+option must be specified if
+.Fl X
+is.
```

Comment:
I'm a bit confused I think; I had an extraneous -X option for the description entries, and have gotten rid of the second one just now (since it doesn't have a -R option).  I've changed that, and the man page looks better to me.

Reply:
What? You.. deleted a synopsis line? Just.. apply this:
```
diff --git a/man/man8/zfs-send.8 b/man/man8/zfs-send.8
index abc83d2d8..397db5063 100644
--- a/man/man8/zfs-send.8
+++ b/man/man8/zfs-send.8
@@ -47,6 +47,7 @@
 .Cm send
 .Op Fl DLPcensvw
 .Op Fl i Ar snapshot Ns | Ns Ar bookmark
+.Ar filesystem Ns | Ns Ar volume Ns | Ns Ar snapshot
 .Nm zfs
 .Cm send
 .Fl -redact Ar redaction_bookmark
@@ -73,6 +74,7 @@
 .Nm zfs
 .Cm send
 .Op Fl DLPRbcehnpvw
+.Op Fl X Ar dataset Ns Oo , Ns Ar dataset Oc Ns ...
 .Op Oo Fl I Ns | Ns Fl i Oc Ar snapshot
 .Ar snapshot
 .Xc
```

Reply:
As I said, confused.